adhocracy-plus/assets/scss/components/_progress.scss

```diff
@@ -1,47 +1,60 @@
-.progress__background,
-.progress__value {
-    stroke-width: 3;
-    fill: none;
-}
+.progress-line {
+    list-style: none;
+    padding: $padding 0 $padding 2*$padding;
 
-.progress__background {
-    stroke: $bg-tertiary;
-}
+    &__entry {
+        display: flex;
+        color: $text-color-gray;
+        margin-top: 4*$padding;
+        position: relative;
 
-.progress__value {
-    stroke: $brand-primary;
-    transform: rotate(-90deg);
-    transform-origin: center;
-}
+        &:first-child {
+            padding-top: 0;
+        }
+    }
 
-.progress__label {
-    text-anchor: middle;
-    font-size: 6px;
-}
+    &__entry--active {
+        color: $brand-tertiary;
+    }
+
+    &__entry--published {
+        display: flex;
+        position: relative;
+        font-weight: bold;
+        color: $text-color;
+    }
+
+    &__icon {
+        color: $border-color;
+    }
+
+    &__icon--active {
+        color: $brand-tertiary;
+        background-color: $body-bg;
+    }
 
-.progress__check {
-    fill: $brand-primary;
+    i {
+        margin-right: $em-spacer;
```

Comment:
@phillimorland this is where the em-spacer is used. I based a lot on mB, so I guess this is where I had it from. I just started reading into em vs rem.... I could replace it with the spacer. What do you think?

Reply:
yeah I would say use our usual $spacer

Reply:
Done!